package.json

```diff
@@ -30,7 +30,7 @@
   "author": "Miller Medeiros <http://blog.millermedeiros.com>",
   "license": "MIT",
   "dependencies": {
-    "esprima": "^2.1"
+    "esprima": "~2.7.1"
```

Comment:
I bump this to the latest and have it locked with `~`, mainly to prevent a similar problem (if an additive feature but backward compatible change is introduced that may break the test). Let me know if this should be changed to a different one.